flang/include/flang/Lower/FIRBuilder.h

```diff
@@ -74,8 +74,9 @@ class FirOpBuilder : public mlir::OpBuilder {
   /// Safely create a reference type to the type `eleTy`.
   mlir::Type getRefType(mlir::Type eleTy);
 
-  /// Create a 1-dimensional sequence of `eleTy` of unknown size.
-  mlir::Type getVarLenSeqTy(mlir::Type eleTy);
+  /// Create a sequence of `eleTy` of dimension rank with unknown extents.
+  /// Default to 1 dimension if rank is not provided.
+  mlir::Type getVarLenSeqTy(mlir::Type eleTy, int rank = 1);
```

Comment:
I'm assuming the idea is to add `rank` number of `?`, so given a type `T`, this returns `!fir.array<?x ... T>`.

Reply:
Yes, thanks, that's a clearer description.